src/util/amount.rs

```diff
@@ -1085,7 +1085,7 @@ pub mod serde {
                     where
                         S: de::SeqAccess<'de>,
                     {
-                        let mut amt_vec = Vec::with_capacity(size_hint::cautious(seq.size_hint()));
+                        let mut amt_vec = Vec::with_capacity(size_hint::cautious::<usize>(seq.size_hint()));
```

Comment:
this should be `::<Amount>`

this applies to the other usages of size_hint

Reply:
> this should be `::<Amount>`
> 
> this applies to the other usages of size_hint

Sorry, I didn't see them, I'll close this then, also can you please check on the monero-rpc-rs issues I made?